bld/CMakeLists.txt

```diff
@@ -66,11 +66,13 @@ endif()
 
 if (CMAKE_VERSION VERSION_LESS "3.1")
```

Comment:
Debian 8.7 provides gcc version 4.9.2 (Debian 4.9.2-10).
But I believe, the statement above checks for version of cmake, which is 3.0.2.
Do we need to check the version of gcc or cmake here?